pyiron_base/storage/datacontainer.py

```diff
@@ -451,50 +439,53 @@ def _read_only_error(cls):
             "finished.".format(cls.__name__)
         )
 
-    def to_builtin(self, stringify=False):
+    def to_builtin(self, stringify=False, load_stubs=True, _no_str_keys=False):
```

Comment:
`load_stubs` should have a short documentation even though it's fairly obvious.  If I see correctly `_no_str_keys` is not used.

Reply:
Yes, documentation is missing and I forgot to pass the `_no_str_keys` to the sub groups. However, it is used and switches off the key conversion such that the keys stay e.g. int if they are.